src/config/default.yml

```diff
@@ -1,20 +1,20 @@
-schemaVersion: 1.0.1.5
-releaseVersion: 5.2.0
+schemaVersion: 1.0.1.6
```

Comment:
I think I saw 1.0.1.7 go thru today - is that for later? :)

Reply:
Ha! Yes, it should be 1.0.1.7.